docs/Gemfile.lock

```diff
@@ -0,0 +1,201 @@
+GEM
```

Comment:
Don't think we should need this file? Not familiar with ruby but '.lock' files don't usually need tracking.